### ISSUE TITLE
Custom Toolbar for "upload-image" doesn't appear to work

### ISSUE THREAD

**Issue by elixirgraphics** [Ionaru/easy-markdown-editor#230]
**Describe the bug**
I'm working to set a custom icon for the Upload Image button. When I do so the button no longer functions. Since the action was not documented I dug around and sussed out that it should be `EasyMDE.drawUploadedImage` and set it as such.

My custom toolbar code is as follows:

```
toolbar: [{
                name: "upload-image",
                action: EasyMDE.drawUploadedImage,
                className: "icon-photo",
                title: "Upload Image",
            }],
```
**Expected behavior**
Should trigger the file chooser to find a local file to upload.

**Comment by elixirgraphics** [comment]
Looks like the binding for Upload Image is missing:

```
EasyMDE.drawUploadedImage = drawUploadedImage;
```

**Comment by Ionaru** [comment]
Yes, the `drawUploadedImage` function is missing from the exports.

You can also use the upload-image button like this: `toolbar: ['upload-image']`.

**Comment by elixirgraphics** [comment]
Thanks. I have definitely gotten the Upload Image to work with your suggested method. I wanted to customize all of the icons with a different font icon set so it was necessary for me to add the above binding to get it to work. Thanks for your help!  :-)

**Comment by Ionaru** [comment]
Added in b4619790671876fce3689b84d5f0e65e033b7c9b.

**Comment by jjxtra** [comment]
How to just change the image button that exists to become an upload image button?
